packages/nimble-components/src/patterns/error/specs/checkbox-radio-error.md

```diff
@@ -0,0 +1,65 @@
+# Error State on Checkbox and Radio Group
+
+## Overview
+
+The `nimble-checkbox` and the `nimble-radio-group` need to have error states associated with them. Following the existing patterns within nimble, this error state will be a red exclamation mark icon next to the component and red error text below the component.
+
+### Background
+
+-   [Visual design spec](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=1295-63148&node-type=canvas&t=sJ2Fhm1vLoZ4zpsK-0)
+-   [Nimble issue 2018: Error state for nimble-checkbox](https://github.com/ni/nimble/issues/2018)
+-   [Nimble issue 2019: Error state for nimble-radio-group](https://github.com/ni/nimble/issues/2019)
+
+Related issues that can and/or should be resolved as part of this effort:
+
+-   [Nimble issue 2091: nimble-checkbox has incorrect height](https://github.com/ni/nimble/issues/2091)
+-   [Nimble issue 2090: Error icon is incorrectly spaced in nimble-text-field](https://github.com/ni/nimble/issues/2090)
+
+## Design
```

Comment:
Currently, both checkboxes and radio buttons respond to clicks (and style for hover) anywhere in the component. If we change the widths to `100%`, that will pose a problem, as (I believe) ARIA guidelines say that only the control and its label should respond to clicks (not blank areas within the component's bounds). I think the same applies to hover styling.

Should have some notes about any changes that we'll need to make.

Reply:
I've updated the HLD with these details, including some screenshots that show where the interactable part of the control is.